notebooks/byod.py

```diff
@@ -32,14 +36,33 @@
     | --------- | ---------  | --------- |
     | NWD1      | NWD1.cram  | NWD1.crai |
     | NWD2      | NWD2.cram  | NWD2.crai |
+
+    ## Assumptions
+    * You are not trying to overwrite a data table that already exists
+    * Your files follow a naming convention either like this...
+    NWD119844.CRAM
+    NWD119844.CRAM.CRAI
+    ...or this:
+    NWD119844.CRAM
+    NWD119844.CRAI
+
+    Files that lack the extension .cram or .crai will not be added to the data table.
```

Comment:
nit: I feel that this is implied.

Reply:
I'm of two minds about this one... @bethsheets, your thoughts? It may be implicit, but a researcher might also be unsure if txt files (common if someone has a lot of data) will "mess up" the table.